src/input/pointer.js

```diff
@@ -314,8 +383,8 @@
     function updateCoordFromEvent(event) {
         var local;
 
-        // reset the touch array cache
-        changedTouches.length = 0;
+        // touch array cache
+        var changedTouches = [];
```

Comment:
I would rather keep this one declared "globally" as before to avoid creating ana array object each time the function is called (this is why it was declared global before)